### DISCUSSION TITLE
Capstone API - POST response 200 tapi res.text Error + data tidak ter-upload di db

### DISCUSSION THREAD

**Discussion by oden-ao** [teamalgoritma/community#861] [Q&A] [answered]
![1](https://github.com/teamalgoritma/community/assets/101117691/050d437a-f657-4746-bb23-d3a801bd048d)
![image](https://github.com/teamalgoritma/community/assets/101117691/99f602c2-7d38-45e4-a1ff-848d343736d4)
Saat mencoba untuk run kode di atas, data tidak muncul di db, response 200 namun res.text error. Tidak ada data duplikat pada db, dan tidak ada error juga di terminal

Berikut adalah kode pada app.py:
![image](https://github.com/teamalgoritma/community/assets/101117691/7a156227-2683-47d0-ba54-80e8564deabd)
![image](https://github.com/teamalgoritma/community/assets/101117691/c0fd7231-89e1-49f9-95c2-953896824b2a)

Dimohon bantuannya, terima kasih!

**Comment by oden-ao** [comment, marked as answer]
Setelah restart notebook dan app.py tampaknya bisa berjalan lagi dengan normal dan speed upload iteration yang cukup cepat, walau masih kurang yakin asal dari masalah tersebut dari mana; Kemungkinan besar karena sudah banyak kode yang di run di Jupyter Notebooknya. Mungkin bisa dijadikan future reference: Restart notebook dan app.py, lalu hanya run kode yang diperlukan/jangan di run above cell (cukup run import dan read csv saja)

**Reply by IrfanChairurrachman** [reply]
Dear @oden-ao ,

Terima kasih atas masukannya. Jika dilihat memang sepertinya tidak ada kesalahan kode yang dimiliki baik di app.py dan di notebook ketika ingin memasukkan data. 

Bisa dikatakan permasalahannya sudah solve dengan cara restart notebook dan app.py nya ya Bu?

**Reply by oden-ao** [reply]
Untungnya iya kak, saya akan mark as answer
